### PR TITLE
Add GitHub PR checks

### PR DESCRIPTION
**Pull request by lfdebrux**
We want to run tests before merging any changes to this repo.

This commit adds a GitHub Actions workflow to run tests (currently only linting).

### CODE REVIEW

**Comment by lfdebrux** [comment]
I've made tests against 3.6 required for merging.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: Test
```
Comment (gidsg) [inline .github/workflows/test.yml:1]: Could we use the template from https://github.com/alphagov/.github/blob/master/workflow-templates/digitalmarketplace-python-ci.yml ? We have updated the cache key and restore logic.

Comment (lfdebrux) [reply .github/workflows/test.yml:1]: Ah, good point

Comment (lfdebrux) [reply .github/workflows/test.yml:1]: Done

**Review by gidsg — APPROVED** [review]
:octocat: 👍

**Pull request merged**